test/Interop/Cxx/foreign-reference/frt-retained-unretained-attributes-error.swift

```diff
@@ -0,0 +1,10 @@
+// RUN: rm -rf %t
+// RUN: not %target-swift-frontend -typecheck -I %S/Inputs  %s -cxx-interoperability-mode=upcoming-swift 2>&1 | %FileCheck %s
+
+import FunctionsAndMethodsReturningFRT
+
+// Testing diagnostic for Global/free C++ functions returning FRT returning FRT with both attributes swift_attr("returns_unretained") and swift_attr("returns_retained")
+let frtLocalVar1 = global_function_returning_FRT_with_both_attrs_returns_retained_returns_unretained(); // CHECK: error: 'global_function_returning_FRT_with_both_attrs_returns_retained_returns_unretained' cannot be annotated with both swift_attr('returns_retained') and swift_attr('returns_unretained') attributes
+
+// Testing diagnostic for C++ Struct having static methods returning FRT returning FRT with both attributes swift_attr("returns_unretained") and swift_attr("returns_retained")
+let frtLocalVar2 = StructWithStaticMethodsReturningFRTWithBothAttributesReturnsRetainedAndReturnsUnretained.StaticMethodReturningFRT();; // CHECK: error: 'StaticMethodReturningFRT' cannot be annotated with both swift_attr('returns_retained') and swift_attr('returns_unretained') attributes
```

Comment:
Typo: double `;;`.
Very minor, but Swift doesn't require `;` at the end of the lines, could you please remove the semicolons?